janusgraph-berkeleyje/src/test/java/org/janusgraph/BerkeleyJanusGraphDatabaseProvider.java

```diff
@@ -0,0 +1,74 @@
+package org.janusgraph;
```

Comment:
Please add the standard copyright header at the top of this and other new files.

Reply:
I see this was done.